vm/src/obj/objstr.rs

```diff
@@ -1104,4 +1139,34 @@ mod tests {
             assert!(!PyString::from(s).istitle(&vm));
         }
     }
+
+    #[test]
+    fn str_translate() {
```

Comment:
We also add python snippets under `tests/snippets`

Reply:
> Are you using the latest rust toolchain?

You are right my default toolchain is stable which has different `rustfmt` than  `nightly`. Now formatting with `cargo +nightly fmt` and it seems all formatting problemsa are fixed now!